Framework/include/QualityControl/Activity.h

```diff
@@ -67,12 +68,15 @@ class Activity
 
   virtual ~Activity() = default;
 
+  void importFromRun(const std::shared_ptr<o2::bookkeeping::Run>& run);
+
   int mId{ 0 };
   int mType{ 0 };
   std::string mPeriodName{};
   std::string mPassName{};
   std::string mProvenance{ "qc" };
   ValidityInterval mValidity{ gFullValidityInterval };
+  std::string mBeamType{};
```

Comment:
We could actually take the opportunity to enforce enumerated beam types. It would be risky to require from the users to know if the string is going to be "PROTON-PROTON" or "pp", or "p-p".

Reply:
I am not sure that we should do it on our side. Rather in BK

Reply:
Whoever does it, I think that the user code should not rely on comparing hardcoded strings.

Reply:
Somewhere, maybe in `Common` or in `AliceO2` we could have some class to define the collision system. Maybe there is one already? (cc @shahor02 ?)